Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocAutoCompleteReferenceValueControlObject.cs

```diff
@@ -57,17 +58,17 @@ private class SelectAutoCompleteAction : WebTestAction
 
       private string _actionName = "BocAutoCompleteReferenceValueControlObject_SelectAutoCompleteAction";
 
-      [NotNull]
+      [JetBrains.Annotations.NotNull]
```

Comment:
This feels noisy. would a 
`using NotNullAttribute = Jetbrains.Annotations.NutNullAttribute`
be better in terms of readability / noise?